### PR TITLE
Turn Whatever: Put this non-deterministic stuff to bed

### PR DESCRIPTION
**Pull request by MadMub**
and get to the meat and potatoes of this 1 million course meal

### CODE REVIEW

File: game.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const username = process.env["USER"]
 
 function start() {
   Math.random = function() {
-
+    return 0.42;
```
Comment (patvice) [inline game.js:10]: :trollface:

**Comment by patvice** [comment]
that ascii!

**Comment by epixa** [comment]
Was this... just skipped?

**Comment by MadMub** [comment]
:( i made that ascii by hand....

**Comment by celsodantas** [comment]
> :( i made that ascii by hand....

damn. impressed. lol

**Pull request merged**